cmd/utils/artifactUtils/compositeAppUtils.go

```diff
@@ -25,15 +28,20 @@ func (compositeApps *CompositeAppList) GetDataIterator() <-chan []string {
 	ch := make(chan []string)
 
 	go func() {
-		for _, compositeApp := range compositeApps.CompositeApps {
-			ch <- []string{compositeApp.Name, compositeApp.Version}
+		for _, compositeApp := range compositeApps.ActiveCompositeApps {
+			ch <- []string{compositeApp.Name, compositeApp.Version, "Active"}
+		}
+		for _, compositeApp := range compositeApps.FaultyCompositeApps {
+			ch <- []string{compositeApp.Name, compositeApp.Version, "Faulty"}
 		}
 		close(ch)
 	}()
 
 	return ch
 }
 
+
 func (compositeApps *CompositeAppList) GetCount() int32 {
-	return compositeApps.Count
+	return compositeApps.ActiveCount + compositeApps.ActiveCount;
```

Comment:
why we add ActiveCount twice ? shouldn't it be TotalCount ?

Reply:
yes it should be. will fix it